src/renderer/actions/network.ts

```diff
@@ -40,3 +46,18 @@ export const saveSelectedNetworkAction = (network: string): ISaveSelectedNetwork
     type: NetworkActionTypes.SELECT_NETWORK,
     payload: network,
 });
+
+export const saveBeaconNodeAction = (url: string, network?: string) => {
+    return async (dispatch: Dispatch<Action<unknown>>, getState: () => IRootState): Promise<void> => {
+        const localDockerName = network ? BeaconChain.getContainerName(network) : undefined;
```

Comment:
maybe use null?
What do we do if it's undefined/null?

Reply:
It's an empty string if it is a remote beacon node. I put network here because of the container name :/

Reply:
What if it's a local node but container is down or offline?

Reply:
This is saved after starting the docker container. We don't have resuming container work anywhere for now